wagtail/wagtailcore/models.py

```diff
@@ -763,13 +763,20 @@ def move(self, target, pos=None):
         logger.info("Page moved: \"%s\" id=%d path=%s", self.title, self.id, new_url_path)
 
     def copy(self, recursive=False, to=None, update_attrs=None, copy_revisions=True, keep_live=True, user=None):
-        # Make a copy
-        page_copy = Page.objects.get(id=self.id).specific
-        page_copy.pk = None
-        page_copy.id = None
-        page_copy.depth = None
-        page_copy.numchild = 0
-        page_copy.path = None
+        """
+        Copy of a given page instance based on a fresh instance to enforce proper multiple inheritance
```

Comment:
This docstring probably wouldn't make much sense to someone reading the docs, perhaps it should be a comment instead